src/App.js

```diff
@@ -0,0 +1,82 @@
+import GlobalStyle from "./globalStyles";
+import Navbar from "./Navbar";
+import Homapage from "./Homapage";
+import Profile from "./Profile";
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+
+import { initializeApp } from "firebase/app";
+import { getStorage } from "firebase/storage";
+import {
+  getFirestore,
+  collection,
+  onSnapshot,
+  setDoc,
+  doc,
+  Timestamp,
+  getDocs,
+  getDoc,
+  addDoc,
+  deleteDoc,
+} from "firebase/firestore";
+
+// TODO: Add SDKs for Firebase products that you want to use
+// https://firebase.google.com/docs/web/setup#available-libraries
+
+// Your web app's Firebase configuration
+// For Firebase JS SDK v7.20.0 and later, measurementId is optional
+const firebaseConfig = {
+  apiKey: "AIzaSyAhSbQw18_zFQ7GzjqhjeAdNeDYl10udoY",
+  authDomain: "tz-tz-fa8a7.firebaseapp.com",
+  projectId: "tz-tz-fa8a7",
+  storageBucket: "tz-tz-fa8a7.appspot.com",
+  messagingSenderId: "325929538936",
+  appId: "1:325929538936:web:982fb0931f435969f4e48d",
+  measurementId: "G-F45EQNSF22",
+};
```

Comment:
Please move into env file...

Reply:
Have the firebaseConfig arranged in the constantFile